docs/upgrade.md

```diff
@@ -45,6 +45,20 @@ for detail on upgrading to Marionette 3. This technique works in both Marionette
 
 The `childEvents` attribute was renamed to `childViewEvents`.
 
+Child view events now only bubble a single level to their direct parent. To
```

Comment:
In that case, would it be more accurate to state that "Child event bubbling above one level is now removed in most instances and deprecated pending removal in others. This can no longer be relied upon." perhaps?